plenum/common/messages/fields.py

```diff
@@ -335,10 +338,14 @@ class TieAmongField(FieldBase):
     _base_types = (list, tuple)
     _length = 2
 
+    def __init__(self, max_length: int, **kwargs):
+        super().__init__(**kwargs)
+        self._max_length = max_length
```

Comment:
Why don't we use LimitedLengthStringField as base class here as well?

Reply:
this type is complex and does not have str in base types, not sure it is good to inherit from string type